README.md

```diff
@@ -34,6 +41,49 @@ module "acm" {
 * [Complete example with DNS validation (recommended)](https://github.com/terraform-aws-modules/terraform-aws-acm/tree/master/examples/complete-dns-validation)
 * [Complete example with EMAIL validation](https://github.com/terraform-aws-modules/terraform-aws-acm/tree/master/examples/complete-email-validation)
 
+## hosted zone in another AWS account
+
+In certain scenarios if hosted zone is present in another AWS account, then we can pass the a different aws provider to the module.
+
+### hosted zone in same account
+
+```hcl
+module "acm" {
+  source  = "terraform-aws-modules/acm/aws"
+  version = "~> v2.0"
+
+  # ... omitted
+
+  providers = {
+    aws = aws
+    aws.route53 = aws
+  }
+```
+
+### hosted zone in different account
+
+```hcl
+provider "aws" {
+  # ... omitted
+}
+
+provider "aws" {
+  alias = "production"
+  # ... omitted
+}
+
+module "acm" {
+  source  = "terraform-aws-modules/acm/aws"
+  version = "~> v2.0"
```

Comment:
If there's a breaking change in this PR, won't this need to be `v3.0`?

Reply:
Yes, agreed! This need to be `v3.0`. I am not aware of the release process, thus kept as it is.